lib/uploadcare.rb

```diff
@@ -51,4 +54,16 @@ module Uploadcare
   setting :framework_data,            default: ''
   setting :file_chunk_size,           default: 100
   setting :logger,                    default: Logger.new($stdout)
+  setting :cdn_base,                  default: ENV.fetch('UPLOADCARE_CDN_BASE', 'https://ucarecdn.com/')
+  setting :custom_cdn_base,           default: ENV.fetch('UPLOADCARE_CUSTOM_CDN_BASE', 'https://ucarecd.net/')
```

Comment:
this is not "custom cdn", this is the new default. this will be confusing for "custom CNAME" users
consider renaming to `cdn_base_postfix` or something alike